src/WormholeConnect.tsx

```diff
@@ -14,17 +14,25 @@ import type { WormholeConnectConfig } from './config/types';
 import type { WormholeConnectTheme } from 'theme';
 import { RouteProvider } from './contexts/RouteContext';
 import { TokensProvider } from './contexts/TokensContext';
+import { WalletProvider } from './contexts/WalletContext';
+import type { WormholeConnectWalletProvider } from './utils/wallet/types';
```

Comment:
Is this type name too verbose? Prefixing with `WormholeConnect` to be consistent with the other props